src/core/server/ui_settings/ui_settings_client.ts

```diff
@@ -228,7 +229,8 @@ export class UiSettingsClient implements IUiSettingsClient {
   }: ReadOptions = {}): Promise<Record<string, any>> {
     try {
       const resp = await this.savedObjectsClient.get<Record<string, any>>(this.type, this.id);
-      return resp.attributes;
+      const attributes = this.translateChanges(resp.attributes, 'timelion', 'timeline');
+      return attributes;
```

Comment:
Can't we just return the result of the translation instead of creating the `attributes` variable?